components/access-management/kfam/bindings_test.go

```diff
@@ -0,0 +1,72 @@
+package kfam
+
+import (
+	"testing"
+
+	rbacv1 "k8s.io/api/rbac/v1"
+)
+
+// Building Binding Object from k8s.io/api/rbac/v1
+func getBindingObject(binding string) *Binding {
+
+	return &Binding{
+		User: &rbacv1.Subject{
+			Kind: "user",
+			Name: binding,
+		},
+		RoleRef: &rbacv1.RoleRef{
+			Kind: "clusterrole",
+			Name: "edit",
+		},
+	}
+
+}
+
+//Table driven tests
+var tests = []struct {
+	in       *Binding
+	out      string
+	hasError bool
+}{
+	{getBindingObject("lalith.vaka@zq.msds.kp.org"), "user-lalith-vaka-zq-msds-kp-org-clusterrole-edit", false},
+	{getBindingObject("397401@zq.msds.kp.org"), "user-397401-zq-msds-kp-org-clusterrole-edit", false},
+	{getBindingObject("lalith.397401@zq.msds.kp.org"), "user-lalith-397401-zq-msds-kp-org-clusterrole-edit", false},
+	{getBindingObject("397401.vaka@zq.msds.kp.org"), "user-397401-vaka-zq-msds-kp-org-clusterrole-edit", false},
+	{getBindingObject("i397401@zq.msds.kp.org"), "user-i397401-zq-msds-kp-org-clusterrole-edit", false},
+}
```

Comment:
Since these tests belong to the `TestGetBindingName` function, should we move this variable declaration inside the function instead of being a global one?